provider_shopper/pubspec.yaml

```diff
@@ -1,5 +1,6 @@
 name: provider_shopper
 description: A shopping app sample that uses Provider for state management.
+publis_to: none
```

Comment:
Typo. Once this is fixed I'm guessing you'll have a green build

Reply:
Argh my bad, I shouldn't be checking PRs Friday evening :)